src/services/room-service.js

```diff
@@ -230,6 +234,10 @@ export default class RoomService {
     return updatedRoom;
   }
 
+  async removeAllRoomsMember(memberUserId) {
```

Comment:
Maybe we can rename this (and also the store function) to something like `removeUserFromAllRooms` or `removeMembershipFromAllRooms(ByUserId|ForUser)`? The current name wants to appear like `removeAllRoomMembers` and only by reading it very carefully you realize that it's something different.

Reply:
Yes that was an uninspired name, I changed it to `removeMembershipFromAllRoomsForUser`, thanks!